lib/hanami/assets/configuration.rb

```diff
@@ -15,47 +15,51 @@ module Assets
     class Configuration
       # @since 0.1.0
       # @api private
-      DEFAULT_SCHEME           = 'http'.freeze
+      DEFAULT_SCHEME                          = 'http'.freeze
 
       # @since 0.1.0
       # @api private
-      DEFAULT_HOST             = 'localhost'.freeze
+      DEFAULT_HOST                            = 'localhost'.freeze
 
       # @since 0.1.0
       # @api private
-      DEFAULT_PORT             = '2300'.freeze
+      DEFAULT_PORT                            = '2300'.freeze
 
       # @since 0.1.0
       # @api private
-      DEFAULT_PUBLIC_DIRECTORY = 'public'.freeze
+      DEFAULT_PUBLIC_DIRECTORY                = 'public'.freeze
 
       # @since 0.1.0
       # @api private
-      DEFAULT_MANIFEST         = 'assets.json'.freeze
+      DEFAULT_MANIFEST                        = 'assets.json'.freeze
 
       # @since 0.1.0
       # @api private
-      DEFAULT_PREFIX           = '/assets'.freeze
+      DEFAULT_PREFIX                          = '/assets'.freeze
 
       # @since 0.1.0
       # @api private
-      URL_SEPARATOR            = '/'.freeze
+      URL_SEPARATOR                           = '/'.freeze
 
       # @since 0.1.0
       # @api private
-      HTTP_SCHEME              = 'http'.freeze
+      HTTP_SCHEME                             = 'http'.freeze
 
       # @since 0.1.0
       # @api private
-      HTTP_PORT                = '80'.freeze
+      HTTP_PORT                               = '80'.freeze
 
       # @since 0.1.0
       # @api private
-      HTTPS_SCHEME             = 'https'.freeze
+      HTTPS_SCHEME                            = 'https'.freeze
 
       # @since 0.1.0
       # @api private
-      HTTPS_PORT               = '443'.freeze
+      HTTPS_PORT                              = '443'.freeze
+
+      # @since 0.3.0-add-options-to-javascript-helper
+      # @api private
+      DEFAULT_SUBRESOURCE_INTEGRITY_ALGORITHM = :sha256
```

Comment:
The rest of these are strings, but I think people will use symbols for this value more than strings. (though it'll still work with Strings, since we call `to_s` on this value)